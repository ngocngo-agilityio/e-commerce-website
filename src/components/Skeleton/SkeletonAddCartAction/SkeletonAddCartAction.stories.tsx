import type { Meta, StoryObj } from '@storybook/react';

// Components
import SkeletonAddCartAction from '.';

const meta = {
  title: 'Skeletons/SkeletonAddCartAction',
  component: SkeletonAddCartAction,
  parameters: {
    layout: 'centered',
  },
  tags: ['autodocs'],
} satisfies Meta<typeof SkeletonAddCartAction>;

export default meta;
type Story = StoryObj<typeof meta>;

export const Default: Story = {
  args: {},
};
