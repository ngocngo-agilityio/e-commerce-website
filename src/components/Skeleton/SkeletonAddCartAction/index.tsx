// Libs
import { Skeleton } from '@chakra-ui/react';

const SkeletonAddCartAction = (): JSX.Element => {
  return (
    <>
      <Skeleton w="210px" h="48px" />
      <Skeleton w="165px" h="48px" mt="36px" mb="32px" />
    </>
  );
};

export default SkeletonAddCartAction;
