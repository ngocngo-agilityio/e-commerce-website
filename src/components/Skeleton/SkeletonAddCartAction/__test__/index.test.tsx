// Libs
import { render } from '@testing-library/react';

// Components
import SkeletonAddCartAction from '..';

describe('SkeletonAddCartAction component', () => {
  test('should match snapshot', () => {
    const { container } = render(<SkeletonAddCartAction />);

    expect(container).toMatchSnapshot();
  });
});
