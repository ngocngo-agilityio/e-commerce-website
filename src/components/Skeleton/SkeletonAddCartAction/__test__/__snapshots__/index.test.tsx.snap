// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkeletonAddCartAction component should match snapshot 1`] = `
<div>
  <div
    class="chakra-skeleton css-axgwne"
  />
  <div
    class="chakra-skeleton css-1lw11jq"
  />
</div>
`;

exports[`SkeletonAddCartAction component should match snapshot for SkeletonProductCategories 1`] = `
<div>
  <div
    class="chakra-skeleton css-axgwne"
  />
  <div
    class="chakra-skeleton css-1lw11jq"
  />
</div>
`;
